charts/managed-identity-wallets/values.yaml

```diff
@@ -28,24 +32,47 @@ revocationService:
   clientIssuanceApiUrl: "http://localhost:8080"
 acapy:
   imageName: "bcgovimages/aries-cloudagent"
-  tag: "py36-1.16-1_0.7.4"
-  ledgerUrl: "https://idu.cloudcompass.ca"
-  label: "CatenaXIssuer"
-  logLevel: "INFO"
-  networkIdentifier: "idunion:test"
-  databaseHost: "acapypostgresql"
-  endpointPort: "8000"
-  adminPort: "11000"
-  adminUrl: "http://localhost:11000"
-  secret:
-    apikey: "0"
-    walletseed: "0"
-    dbaccount: "postgres"
-    dbadminuser: "postgres"
-    dbadminpassword: "postgres"
-    dbpassword: "postgres"
-    jwtsecret: "0"
-    walletkey: "0"
+  tag: "py36-1.16-1_0.7.5"
+  # AcaPy endorser instance
+  endorser:
+    ledgerUrl: "https://idu.cloudcompass.ca"
+    label: "CatenaXIssuer"
+    logLevel: "INFO"
+    networkIdentifier: "idunion:test"
+    databaseHost: "acapypostgresql"
+    endpointPort: "8000"
+    adminPort: "11000"
+    adminUrl: "http://localhost:11000"
+    secret:
+      apikey: "0"
+      walletseed: "0"
+      dbaccount: "postgres"
+      dbadminuser: "postgres"
+      dbadminpassword: "postgres"
+      dbpassword: "postgres"
+      jwtsecret: "0"
+      walletkey: "0"
+  # AcaPy multi-tenancy instance
+  mt:
+    ledgerUrl: "https://idu.cloudcompass.ca"
+    label: "CatenaXIssuer"
+    logLevel: "INFO"
+    networkIdentifier: "idunion:test"
+    databaseHost: "acapypostgresql"
+    endpointPort: "8003"
+    adminPort: "11003"
+    adminUrl: "http://localhost:11003"
+    endorserPublicDid: "MhLrwtKpZhNCzazMeofPQH"
```

Comment:
Same here, would have expected an empty value and an explanation in `charts/README.md` file.